pkg/translator/jaeger/constants.go

```diff
@@ -18,7 +18,24 @@ import (
 	"errors"
 )
 
+const (
+	// Integer types use two's complement arithmetic.
+	maxUint = ^uint(0)
+	maxInt  = int64(maxUint >> 1)
+	minInt  = -maxInt - 1
```

Comment:
Why not using "math" package? https://pkg.go.dev/math#pkg-constants `math.MaxInt`?